jhipster-framework/src/main/java/io/github/jhipster/security/ssl/UndertowSSLConfiguration.java

```diff
@@ -37,7 +36,7 @@
  * This can only be activated with HTTPS and a cipher suite defined by the user (server.ssl.ciphers).
  * <p>
  * Please note that when using JHipster, you can use the `server.ssl.ciphers` property that is commented out
```

Comment:
remove the "is commented out" part as it is not commented out anymore.

Reply:
Done